docs-src/guide-rpc-server-api.scrbl

```diff
@@ -0,0 +1,333 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@title[
+  #:version reach-vers
+  #:tag     "guide-rpc-server-api"
+  ]{The Reach RPC server API and how to use it}
+
+The Reach RPC server's design is deliberately lean and simple. In fact, it
+mostly just re-exports everything offered by the standard library via HTTPS
+under a corresponding @urlin{/stdlib/<function name>} route. Arguments and
+their order remain unchanged.
+
+@;TODO better example(?)
+For example, rather than leveraging the standard library directly like so:
+@js{ const balance = stdlib.formatCurrency('100000000000000000000', 18); }
+
+...an equivalent RPC request is issued, where the payload is a JSON-encoded
+array which represents ordered arguments to the target function:
+@js{ POST /stdlib/formatCurrency ['100000000000000000000', 18] }
+
```

Comment:
Then say something like, "This same pattern extends to more intricate operations, like `stdlib.newTestAccount`"